readme.md

```diff
@@ -20,21 +22,49 @@ How to build bsvirtualkb.html
     - bsvirtualkb.json (see below)
 - This also creates "keyboard.zip" in the presentation directory   
     
-bsvirtualkb.json
-----------------
+## bsvirtualkb.json
+
 A config file that currently is only used to pick the languages available in the language dropdown. At present, the 
 only languages supported are English and French.
 
+## The plugin itself
 
-BrightAuthor plugin
--------------------
-There are three example presentations that use the "virtualKeyboard.brs" plugin:
+The script that defines the plugin to be used in BrightAuthor(:Connected) is `virtualKeyboard.brs`. This script is written so that the keyboard is dynamically sized and placed on the screen based on the screen's resolution. It also automatically rotates the keyboard based on the screen orientation selected in your presentation. Important variables in the plugin script are:
 
-- ./presentation/KeyboardPres.bpf
-- ./presentation/KeyboardPortraitBottomOnLeft.bpf
-- ./presentation/KeyboardPortraitBottomOnRight.bpf
+- `startX`: top left x coordinate of the keyboard's rectangle
+- `startY`: top left y coordinate of the keyboard's rectangle
+- `rectX`: x dimension of the keyboard's rectangle
+- `rectY`: y dimension of the keyboard's rectangle
+- `monitorOrientation`: orientation of the screen, set in BA(:Con)
+- `transform`: rotation of the keyboard's rectangle in degrees, in the negative (-) direction by the Right Hand Rule
+- __Note__ `Cint` simply rounds to the nearest integer
+
+## Editing the Virtual Keyboard
+
+There are three important aspects that users of the keyboard may want to change. These aspects are: placement, size, and orientation. All three of these are editted in the plugin script `virtualKeyboard.brs`. Assuming you are using the script provided in this plugin - 
+
+To edit placement:
+- Change the `startX` and `startY` variables defined on lines 35-36
+
+To edit size:
+- Change the `rectX` and `rectY` variables defined on lines 32-33
```

Comment:
To avoid maintaining references to variable's line numbers, the function virtualKeyboard_Initialize() or acknowledgement of variable could be sufficient.

Reply:
smart - will remove line references